ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageManager.java

```diff
@@ -93,6 +103,33 @@ public void setTintColor(ReactImageView view, @Nullable Integer tintColor) {
   }
 
   @Override
+  protected void addEventEmitters(final ThemedReactContext reactContext, final ReactImageView view) {
+    final EventDispatcher mEventDispatcher = reactContext.getNativeModule(UIManagerModule.class)
+        .getEventDispatcher();
+
+    view.setLoadingControllerListener(new BaseControllerListener<ImageInfo>() {
+      @Override
+      public void onFinalImageSet(
```

Comment:
You can also override the `onSubmit` method to send an onLoadStart event, and `onFailure` should also send out onLoadEnd.